.husky/pre-commit

```diff
@@ -0,0 +1 @@
+pnpm lint-staged
```

Comment:
I think this folder should be on the gitignore

Reply:
only the `.husky/_` should not be pushed.

but this file should be committed because it's what `husky` will use to install the git hooks.
(when we run the `prepare` script)